python/spacewalk/server/importlib/backend.py

```diff
@@ -1207,6 +1206,7 @@ def update_channels_affected_by_errata(self, dml):
         for op_type in ["insert", "update", "delete"]:
             op_values = getattr(dml, op_type)
             for table_name, values_hash in list(op_values.items()):
+                field = "id"
                 if table_name == "rhnErrata":
                     field = "id"
                 elif "errata_id" in values_hash:
```

Comment:
Because we have the field var on L#1209, we can simplify this, right?

```suggestion
                if "errata_id" in values_hash:
```

Reply:
This solution is not 100% correct. I made another one.